test/mlir/affine_conversion/callback_ops.mlir

```diff
@@ -4,9 +4,25 @@
 
 // -----
 
+// Convbias Op
```

Comment:
nit: we should rename the folder to something like "lowering" instead of affine_conversion